examples/cpp_sycl/makefile_lnx

```diff
@@ -116,7 +116,7 @@ sointel64:
 _make_ex: $(RES)
 
 vpath
-vpath %.cpp $(addprefix ./source/,datasource covariance linear_regression logistic_regression optimization_solvers pca pca_transform kmeans moments k_nearest_neighbors gradient_boosted_trees)
+vpath %.cpp $(addprefix ./source/,datasource covariance linear_regression logistic_regression optimization_solvers pca pca_transform kmeans moments k_nearest_neighbors gradient_boosted_trees dbscan)
```

Comment:
Don't we need to have this list sorted alphabetically?

Reply:
Almost every method is added at the end for now. All are in historical order, I guess

Reply:
No, at the beginning all methods were sorted: problems started from `k_nearest_neighbors`. It is not critical but should be fixed later for both master and releases